spec/rails_helper.rb

```diff
@@ -88,3 +88,11 @@
     end
   end
 end
+
```

Comment:
This helper removes the need of calling `page.find("body").click` as a separate step when validating textboxes since it forces the textbox to lose focus (via tabbing out)